lib/web_ui/lib/src/engine/platform_dispatcher.dart

```diff
@@ -1320,3 +1318,104 @@ double findBrowserTextScaleFactor() {
   final num fontSize = parseFontSize(domDocument.documentElement!) ?? _defaultRootFontSize;
   return fontSize / _defaultRootFontSize;
 }
+
+class ViewConfiguration {
```

Comment:
I've been thinking of doing `class WebViewConfiguration extends ViewConfiguration` to be able to store some additional web-only view configuration (for example: the embedding strategy that we want to use for a given view).

Do you think it's OK to extend the ViewConfiguration like that, or should I put my additional view configuration in a separate data structure within the web engine?

Reply:
Extending ViewConfiguration should be perfectly fine. I believe in the current setup you could even put the extra data right into the ViewConfiguration class. At this point it is essentially a private class of the web engine that only shares a name with the (also private) class in the non-web engine.

Reply:
This is the perfect place for the EmbeddingStrategy object that I have for each View so it knows how to stamp itself into the DOM. Thanks!